### PR TITLE
t-unziper: Change of dialog for 'Duplicate name prevention'

### LINKED ISSUE #265: t-unzipper: Label for checkbox „Do not prefix symbolic name“ is confusing for normal user
In t-unzipper, there is checkbox „Do not prefix symbolic name“ which is a bit confusing for the normal user (normal user should not know anything about symbolic name).

Proposal: Change the label of the checkbox - in EN, SK. E.g to "Prevencia duplicitných názvov".

### PR DESCRIPTION
**Pull request by mbl-eea-sk**
Resolves #265

### CODE REVIEW

File: t-unzipper/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 |Parameter                                       |Description                                                              |                                                        
 |------------------------------------------------|-------------------------------------------------------------------------|
-|**Do not prefix symbolic name (checkbox):**     |If checked then output symbolic names of output files are not prefixed with symbolic name of unzipped file. Uncheck to prevent symbolic names collision if multiple zip files with same structure are unzipped. If you do not know, then uncheck this. | 
+|**Duplicate name prevention. (checkbox):**     |If checked prevents symbolic names collision if multiple zip files of the same structure are unzipped. Uncheck to avoid symbolic names prefixing. If not sure keep this checkbox checked. |
```
Comment (skrchnavy) [inline t-unzipper/README.md]: This is documentation for user, it shall not contain technical details like symbolic name. 
Something like:
*If checked DPU prevents collision in names of files send on output when multiple zip files with the same/similar structure are unzipped. Uncheck sends on output files names as are in zip files. If not sure keep checked.*

File: t-unzipper/src/main/resources/resources.properties [hunk 2]
```diff
@@ -11,7 +11,7 @@ unzipper.errors.file.outputdir = Can't get base output directory.
 unzipper.errors.file.encrypted = Zip file is encrypted.
 
 # configuration dialog
-unzipper.dialog.unzip.noprefix = Do not prefix symbolic name.
-unzipper.dialog.unzip.noprefix.description = If checked then output symbolic names of output files are not prefixed with symbolic name of unzipped file. Uncheck to prevent symbolic names collision if multiple zip files with same structure are unzipped. If you do not know, then uncheck this.
+unzipper.dialog.unzip.noprefix = Duplicate name prevention.
```
Comment (skrchnavy) [inline t-unzipper/src/main/resources/resources.properties]: also translation keys should correspond the meaning. 
`....notprefix....` to `...duplicity.avoid...

File: t-unzipper/src/main/java/eu/unifiedviews/plugins/transformer/unzipper/UnZipperVaadinDialog.java [hunk 3]
```diff
@@ -32,13 +32,13 @@ public void buildDialogLayout() {
 
     @Override
     protected void setConfiguration(UnZipperConfig_V1 c) throws DPUConfigException {
-        checkNotPrefix.setValue(c.isNotPrefixed());
+        checkNotPrefix.setValue(!c.isNotPrefixed());
```
Comment (skrchnavy) [inline t-unzipper/src/main/java/eu/unifiedviews/plugins/transformer/unzipper/UnZipperVaadinDialog.java]: Documentation in code shall explain logic, why is used inversion of configuration with references to github issue where it was requested.

Comment (mkl-eea-sk) [inline t-unzipper/src/main/java/eu/unifiedviews/plugins/transformer/unzipper/UnZipperVaadinDialog.java]: i will upgrade config class

Comment (skrchnavy) [inline t-unzipper/src/main/java/eu/unifiedviews/plugins/transformer/unzipper/UnZipperVaadinDialog.java]: as dicussed, config class upgrade for current situation is not good solution.

**Pull request merged**